src/patterns/guide/index.njk

```diff
@@ -0,0 +1,66 @@
+---
+group: Pages
+title: Guide
+---
+{% from "views/partials/example/_macro.njk" import patternlibExample %}
+{% from "components/external-link/_macro.njk" import onsExternalLink %}
+{% from "components/panel/_macro.njk" import onsPanel %}
+
+
+The guide is a collection of related pages with features that help the user understand and navigate a topic or subject.
+
+Each of the related pages within a guide contain a [table of contents](/components/toc) that shows all pages in the guide. 
+
+There is an additional navigation component added to a guide. The [pager](/components/pagination#pager) is displayed at the bottom of each page allowing the user to move on to the next (or previous) page.
+
+## When to use this pattern
+
+Use this pattern when you want to present a collection of related pages on a topic.
+
+## How to use this pattern
+
+View the nunjucks output below which shows the components required to create the guide pattern.
+
+The first page of a guide serves as the "parent" to the other pages. In the table of contents it should be linked as "Overview". 
+For accessibility the title `h1` should contain an additional visually hidden `span` that provides the additional "Overview" text e.g.
+
+`<h1 class="u-mb-l">`<br>
+&nbsp;&nbsp;`<span class="u-fs-xxl u-db">About the census </span>`<br>
+&nbsp;&nbsp;`<span class="u-vh">: Overview</span>`<br>
+`</h1>`<br><br>
```

Comment:
Shouldn't we style these codeblocks like other codeblocks for continuity? Like the one on the "Get Started" page

Reply:
I've just attempted to use the same markup, but it doesn't work. It ignores certain tags and I think it needs a convoluted class name combination to get it to work. I'd rather come up with a better solution.